### ISSUE TITLE
Unable to build upstream PP

### ISSUE THREAD

**Issue by cmaoling** [portfolio-performance/portfolio#3468]
**Describe the bug**
Unchanged upstream master branch fails to find JRE:

```
user@effb30daf7fb:/data/portfolio$ git status 
On branch restart-cmaoling
Your branch is up to date with 'upstream/master'.

nothing to commit, working tree clean
user@effb30daf7fb:/data/portfolio$ git remote -v
origin  git@github.com:cmaoling/portfolio.git (fetch)
origin  git@github.com:cmaoling/portfolio.git (push)
upstream        https://github.com/buchen/portfolio.git (fetch)
upstream        https://github.com/buchen/portfolio.git (push)
user@effb30daf7fb:/data/portfolio$ export MAVEN_OPTS="-Xmx4g";  mvn -f pom.xml clean verify
[INFO] ### Using TychoRepositoryTransport for remote P2 access ###
[INFO]     Cache location:         /home/user/.m2/repository/.cache/tycho
[INFO]     Transport mode:         online
[INFO]     Update mode:            cache first
[INFO]     Minimum cache duration: 60 minutes
[INFO]       (you can configure this with -Dtycho.p2.transport.min-cache-minutes=<desired minimum cache duration>)
[INFO] Resolving target definition file:/data/portfolio/portfolio-target-definition/portfolio-target-definition.target for environments=[linux/gtk/x86_64, linux/gtk/aarch64, macPOM file pom.xml specified with the -f/--file command line argument does not exist
[INFO] Scanning for projects...
[...]
[ERROR] Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'portfolio-jre' from location https://www.portfolio-performance.info/bundled-jre/17.0.5:No repository found at https://www.portfolio-performance.info/bundled-jre/17.0.5. -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'portfolio-jre' from location https://www.portfolio-perormance.info/bundled-jre/17.0.5
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:120)

```

**To Reproduce**
1. Checkout master
3. Run export MAVEN_OPTS="-Xmx4g";  mvn -f pom.xml clean verify
5. See error
No repository found at https://www.portfolio-performance.info/bundled-jre/17.0.5. -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'portfolio-jre' from location https://www.portfolio-perormance.info/bundled-jre/17.0.5

**Expected behavior**
Clean build of latest PP version

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Linux, macOS]
 - Version [e.g. 0.47.0]

**Comment by bitwave** [comment]
Same problem with the AUR package for Arch Linux:

```
[ERROR] Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'portfolio-jre' from location https://www.portfolio-performance.info/bundled-jre/17.0.5: No repository found at https://www.portfolio-performance.info/bundled-jre/17.0.5. -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'portfolio-jre' from location https://www.portfolio-performance.info/bundled-jre/17.0.5
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:120)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
```

**Comment by buchen** [comment]
Apparently, by transferring the repositories, it broke the endpoints of not transferred repositories. Also, I cannot point different repositories to the same domain anymore (maybe because it is not the personal account website anymore).

Anyway, I have updated the URLs: ad6450a36ebd833c2ee6bb69fa37f343bcc05a46

I cleared all caches on Github actions. The build succeeded.
https://github.com/portfolio-performance/portfolio/actions/runs/5711338496/job/15472819410
